tethys_apps/models.py

```diff
@@ -229,9 +236,131 @@ def get_value(self, *args, **kwargs):
         raise NotImplementedError()
 
 
+
 class CustomSetting(TethysAppSetting):
+    # value = models.CharField(max_length=1024, blank=True, default="")
+    # default = models.CharField(max_length=1024, blank=True, default="")
+    objects = InheritanceManager()
+    type_custom_setting = models.CharField(max_length=1024, blank=True, default="")
+
+
+class CustomSecretSetting(CustomSetting):
```

Comment:
Rename this to SecretCustomSetting

Reply:
CustomSecretSetting has been renamed SecretCustomSetting